docker-compose.yml.development

```diff
@@ -0,0 +1,51 @@
+# This file is configured for deployment of the CantusDB project in local development.
+# When building the project locally, replace the contents of docker-compose.yml with the
+# contents of this file.
+
+version: '3'
+
+services:
+  django:
+    build:
+      context: ./django
+    volumes:
+      - ./:/code/
+      - static_volume:/resources/static
+      - media_volume:/resources/media
+      - api_cache_volume:/resources/api_cache
+    env_file: ./config/envs/dev_env
+    ports:
+      - 3000:3000
+    restart: always
+    depends_on:
+      - postgres
+
+  nginx:
+    build:
+      context: ./nginx
+    ports:
+      - 80:80
+      # - 443:443
```

Comment:
Now that I think about it, I'm not even sure why we need to change this for development at all?

Reply:
I'll need to check, but I'm pretty sure that if we keep `443:443` here, Nginx will go looking for certificates, fail to find them, and error out.

Reply:
I think nginx will only look for it if there is `ssl` in the `listen` line...but yes, good to check

Reply:
it works with 443 present, and I changed this back (i.e. uncommented in the file).